src/wing/CCPACSWingSection.h

```diff
@@ -37,32 +37,58 @@ class CCPACSWingSection : public generated::CPACSWingSection
 public:
     TIGL_EXPORT CCPACSWingSection(CCPACSWingSections* parent, CTiglUIDManager* uidMgr);
 
-    // Get element count for this section
+    /**
+    * @brief Get element count for this section
+    * @return int
```

Comment:
More of a comment for discussion than a change request: For return values, you would typically add a textual description of the return value after the type of the return value, such as 

```
@retrun int The number of elements in this section
```

Do we need to document the return value for simple getters such as this one? This applies to other docstrings in this PR as well.

Reply:
agreed that we don't add description for obvious return value